src/python/gem5/components/boards/lupv_board.py

```diff
@@ -269,6 +270,10 @@ def generate_device_tree(self, outdir: str) -> None:
         Creates two files in the outdir: 'device.dtb' and 'device.dts'
         :param outdir: Directory to output the files
         """
+        excep_code = { 'INT_SOFT_SUPER': 1, 'INT_TIMER_SUPER': 5,
+                       'INT_TIMER_MACHINE': 7, 'INT_EXT_SUPER': 9,
+                       'INT_EXT_MACHINE': 11 }
+
```

Comment:
Would you be able to declare this dictionary in such a way that both functions have access to it?

Reply:
agreed. I'd put it as a class member